akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -515,7 +513,26 @@ class Http2ServerSpec extends AkkaSpec with WithInPendingUntilFixed with Eventua
     }
 
     "support low-level features" should {
-      "respond to PING frames" in pending
+      "respond to PING frames (spec 6_7)" in new TestSetup with RequestResponseProbes {
+        sendFrame(FrameType.PING, new ByteFlag(0x0), 0, ByteString("data1234")) // ping frame data must be of size 8
```

Comment:
@ktoso in your original PR this was broken afaics because ping frame data needs to have size 8.